sympy/matrices/matrices.py

```diff
@@ -839,7 +842,8 @@ def cholesky_solve(self, rhs):
             L = (self.T*self)._cholesky()
             rhs = self.T*rhs
         else:
-            raise NotImplementedError("Under-determined System.")
+            raise NotImplementedError('Under-determined System. \
```

Comment:
This will be printed with a lot of spaces from the second line. You can use automatic string concatenation [1] to do this, like:
```
some_function("a long string that "
              "wraps to the second line")
```
Similarly for your other exceptions.

[1] https://docs.python.org/3.5/reference/lexical_analysis.html#string-literal-concatenation